src/service/platform-impl.js

```diff
@@ -70,7 +70,7 @@ export class Platform {
    * @return {boolean}
    */
   isFirefox() {
-    return /Firefox/i.test(this.navigator_.userAgent) && !this.isEdge();
+    return /Firefox|FxiOS/i.test(this.navigator_.userAgent) && !this.isEdge();
```

Comment:
I realize we do for Chrome, but we really shouldn't treat Chrome and Firefox on iOS as Chrome and Firefox. I guess the precedent wins here.